dev-libs/liborcus/liborcus-9999.ebuild

```diff
@@ -4,51 +4,56 @@
 EAPI=7
 
 PYTHON_COMPAT=( python3_{6,7,8} )
-inherit python-single-r1
+inherit python-single-r1 autotools
```

Comment:
`autotools` shouldn't be required for release package, why did you change it?

Reply:
wrong assumption, fixed locally.